client/extensions/woocommerce/app/settings/payments/index.js

```diff
@@ -54,8 +85,19 @@ class SettingsPayments extends Component {
 function mapStateToProps( state ) {
 	const site = getSelectedSiteWithFallback( state );
 	return {
+		isSaving: Boolean( getActionList( state ) ),
 		site,
 	};
 }
 
-export default connect( mapStateToProps )( localize( SettingsPayments ) );
+function mapDispatchToProps( dispatch ) {
+	return {
+		actions: bindActionCreators(
```

Comment:
I haven't seen this before. Why not just pass these in directly (and access via `this.props`) like we do elsewhere?